schemas/com.snowplowanalytics.snowplow.storage.rdbshredder/shredding_complete/jsonschema/1-0-0

```diff
@@ -0,0 +1,86 @@
+{
+  "$schema": "http://iglucentral.com/schemas/com.snowplowanalytics.self-desc/schema/jsonschema/1-0-0#",
+  "description": "Schema for information about the shredded data, for loader to read",
+  "self": {
+    "vendor": "com.snowplowanalytics.snowplow.storage.rdbshredder",
+    "name": "shredding_complete",
+    "format": "jsonschema",
+    "version": "1-0-0"
+  },
+  "type": "object",
+  "properties": {
+    "base": {
+      "description": "Blob storage path to the root of the batch",
+      "type": "string",
+      "maxLength": 1024
+    },
+    "compression": {
+      "description": "File compression type",
+      "enum": ["GZIP", "NONE"]
+    },
+    "types": {
+      "description": "Set of Iglu URIs and the format they were shredded into",
+      "type": "array",
```

Comment:
We can add that the minimum length of this array is 1.

Reply:
Why? It's totally okay to not have any shredded types in a batch. Even more, it's totally okay to not have any _events_ in a batch.

Reply:
I was thinking that shredder should not emit a message if it didn't process any data, but actually you're right, it's better to have a message that says 0 than not having a message, as not having a message can indicate a problem.